requirements.txt

```diff
@@ -9,4 +9,4 @@ matplotlib==1.5.3
 numpy>=1.10.1
 owslib==0.9.1
 python-dateutil==2.4.2
-Shapely==1.6.0
+Shapely==1.6.2
```

Comment:
Unless there is a reason to pin to a very specific version, I think all of the entries in requirements.txt should be rewritten as `>=` rather than `==`.